### ISSUE TITLE
Apply multiple transformations results in AttributeError

### ISSUE THREAD

**Issue by tampe125** [KaiyangZhou/deep-person-reid#404]
Hello,

I'm trying to apply some data augmentation to my dataset. I'm using built-in transforms, but I'm applying `k_tfm=2` . Sadly this produces an error in the softmax loss function at this line: https://github.com/KaiyangZhou/deep-person-reid/blob/master/torchreid/engine/image/softmax.py#L82

It makes sense, since when we're applying transformations multiple times, we return a list of images instead of a single one.  
Can you reproduce it? Do you have any suggestions?

**Comment by KaiyangZhou** [comment]
Hi, very sorry for replying this late.

`k_tfm` is used when you need to have, e.g., two batches of images of the same IDs but with different augmentations for consistency training.

You will have to modify the code, or create your own training engines to use this.

Hope this helps.

**Comment by tampe125** [comment]
Ok, thank you for your time
